src/super_gradients/training/models/optical_flow_models/raft/raft_variants.py

```diff
@@ -0,0 +1,140 @@
+import copy
+from typing import Union, Optional, Callable
+
+import torch
+import torch.nn as nn
+from omegaconf import DictConfig
+
+from super_gradients.common.object_names import Models
+
+from super_gradients.common.registry import register_model
+from super_gradients.module_interfaces import SupportsReplaceInputChannels
+from super_gradients.module_interfaces.exportable_optical_flow import ExportableOpticalFlowModel
+from super_gradients.training.models import get_arch_params, SgModule
+from super_gradients.training.utils.utils import HpmStruct, get_param
+
+from .raft_base import Encoder, ContextEncoder, FlowIterativeBlock
+
+
+class RAFT(ExportableOpticalFlowModel, SgModule):
```

Comment:
Please include a docstring with a link to original paper/implementation

Reply:
And add docstrings to the rest of the methods of this class.